Source/WebCore/accessibility/AccessibilityNodeObject.cpp

```diff
@@ -1224,11 +1224,15 @@ static RefPtr<Element> nodeActionElement(Node& node)
 {
     auto elementName = WebCore::elementName(node);
     if (RefPtr input = dynamicDowncast<HTMLInputElement>(node)) {
-        if (!input->isDisabledFormControl() && (input->isRadioButton() || input->isCheckbox() || input->isTextButton() || input->isFileUpload() || input->isImageButton()))
+        if (!input->isDisabledFormControl() && (input->isRadioButton() || input->isCheckbox() || input->isTextButton() || input->isFileUpload() || input->isImageButton() || input->isTextType()))
             return input;
     } else if (elementName == ElementName::HTML_button || elementName == ElementName::HTML_select)
         return &downcast<Element>(node);
 
+    // Content editable nodes should also be considered action elements, so they can accept presses.
+    if (HTMLElement::editabilityFromContentEditableAttr(node, PageIsEditable::No) != Node::Editability::ReadOnly)
```

Comment:
This is a new ancestor walk. Is that necessary, or can we just check for the `contenteditable` attribute?